lib/google/gax/grpc/status_details.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016, Google Inc.
```

Comment:
New files, update to Copyright 2019 Google LLC

Reply:
Updated.